infra/gcp/ensure-main-project.sh

```diff
@@ -313,6 +313,51 @@ EOF
     read -rs
 }
 
+# Eventually we would like to use kubernetes-external-secrets to manage
+# all secrets in aaa; not sure how far we are on that. So for now, at least
+# ensure that the existing kubernetes-public secrets created for humans
+# to manually sync into the aaa cluster are managed by this script.
```

Comment:
@spiffxp kubernetes-external-secrets is up and running. Before we switch to `ExternalSecret`, for each secret created in kubernetes-public we need to add a new version containing only the value of the `data` field instead of the whole Kubernetes Secret.

Reply:
I opened https://github.com/kubernetes/k8s.io/issues/2091, I couldn't find an umbrella issue for this at a glance but may have missed it